apps/nextjs/next.config.js

```diff
@@ -161,37 +159,41 @@ const getConfig = async (phase) => {
 
 module.exports = getConfig;
 
-module.exports = withSentryConfig(module.exports, {
-  // For all available options, see:
-  // https://github.com/getsentry/sentry-webpack-plugin#options
+if (!process.env.TURBOPACK) {
```

Comment:
I can see that there's a benefit for not having the monitoring instrumentation in here. But what if we need to test sentry locally?

Reply:
In that case, running the server in non-turbo dev mode would let you test it

Reply:
The benefit is just to try to remove the sentry / turbo incompatibility

Reply:
I think that we should only have the turbo mode as default, to be honest